content/es/about.md

```diff
@@ -10,18 +10,17 @@ menu:
     parent: about
 ---
 
-Let's Encrypt es una autoridad de certificación (AC, o CA por sus siglas en inglés) gratuita, automatizada, y abierta, manejada para el beneficio público. Es un servicio proveido por el [Internet Security Research Group (ISRG)](https://abetterinternet.org/).
+<em>Let's Encrypt</em> es una autoridad de certificación (AC, o <em>CA</em> por sus siglas en inglés) gratuita, automatizada, y abierta, que existe para el beneficio del público. Es un servicio provisto por el [Internet Security Research Group (ISRG)](https://abetterinternet.org/).
```

Comment:
I think it's better to use Markdown syntax: https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#emphasis
```suggestion
*Let's Encrypt* es una autoridad de certificación (AC, o *CA* por sus siglas en inglés) gratuita, automatizada, y abierta, que existe para el beneficio del público. Es un servicio provisto por el [Internet Security Research Group (ISRG)](https://abetterinternet.org/).
```